### PR TITLE
mention logging requirement in DEVELOP.rst

### PR DESCRIPTION
**Pull request by celaus**

### CODE REVIEW

File: DEVELOP.rst [hunk 1]
```diff
@@ -88,11 +88,17 @@ To compile run::
 
     ./gradlew compileJava
 
+Before running, a config file for log4j is required. Create an empty default with::
+
+    touch config
+
+(For individual configuration see http://logging.apache.org/log4j/1.2/faq.html#noconfig)
```
Comment (chaudum) [inline DEVELOP.rst]: can you make a link out of the URL http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#hyperlink-targets
also: are there any useful defaults to show here?

File: DEVELOP.rst [hunk 2]
```diff
@@ -88,11 +88,17 @@ To compile run::
 
     ./gradlew compileJava
 
+Before running, a config file for elasticsearch's log4j logging is required. Create an empty default with::
+
+    touch config
+
+If you want to configure elasticsearch or its logging, see the `Elasticsearch reference`_ .
```
Comment (chaudum) [inline DEVELOP.rst]: pls dont refer to ES documentation, rather use: [https://crate.io/docs/en/stable/configuration.html#logging](https://crate.io/docs/en/stable/configuration.html#logging)

Comment (chaudum) [inline DEVELOP.rst]: **or** its logging ?

**Comment by mfussenegger** [comment]
ack

**Pull request merged**